packages/run-protocol/src/econ-behaviors.js

```diff
@@ -574,14 +568,14 @@ export const startRunStake = async (
       produce: { Attestation: attestationIssuerR },
     },
   },
-  config = {
-    debtLimit: 1_000_000_000_000n,
-    mintingRatio: [1n, 4n],
-    interestRateBP: 250n,
-    loanFeeBP: 200n,
-    chargingPeriod: SECONDS_PER_HOUR,
-    recordingPeriod: SECONDS_PER_DAY,
-  },
+  {
+    debtLimit = 1_000_000_000_000n,
+    mintingRatio = [1n, 4n],
+    interestRateBP = 250n,
+    loanFeeBP = 200n,
+    chargingPeriod = SECONDS_PER_HOUR,
+    recordingPeriod = SECONDS_PER_DAY,
+  } = {},
```

Comment:
I take it that the intent is if a partial object is provided that its values will come through and the parts that aren't defined will get these defaults. I haven't see that before so I'll trust that you have confirmed that works.

Reply:
Yes that's how default assignment in any destructure works. I've used it plenty of times, can confirm it works :)